lumicks/pylake/detail/confocal.py

```diff
@@ -390,6 +391,16 @@ def export_tiff(self, filename, *, dtype=np.float32, clip=False):
             If enabled, the photon count data will be clipped to fit into the desired `dtype`.
             This option is disabled by default: an error will be raise if the data does not fit.
         """
+        super().export_tiff(filename, dtype=dtype, clip=clip)
+
+    def _tiff_frames(self, iterator=False) -> npt.ArrayLike:
+        """Create frames of TIFFs used by `export_tiff().`"""
+        image = self.get_image()
+        return image if image.ndim >= 4 else np.expand_dims(image, axis=0)
```

Comment:
do we actually need the `expand_dims` still? did you test without?

Reply:
Yes, we need `expand_dims`, as `export_tiff` iterates over all frames of the imagestack. Without `expand_dims`, `export_tiff` of one frame scans (and kymos) would try to iterate over the rows of the image zipped together with the timestamp range(s) and consequently would story only one image row.